en/rest/hooks.mdown

```diff
@@ -0,0 +1,607 @@
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/triggers/Game/beforeSave', json.dumps(
+      {"url": "https://api.example.com/Game/_beforeSave"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{
+  "className": "Game",
+  "triggerName": "beforeSave",
+  "url": "https://api.example.com/Game/_beforeSave"
+}
+```
+
+It returns the class name, trigger name and url of the modified trigger webhook.
+
+If you try to update a trigger webhook and a cloud code trigger with the same name already exists, upon successful update the response json has an additional `warning` field informing about the name conflict. Note that, trigger webhooks takes precedence over cloud code triggers.
+
+For example,
+```bash
+curl -X PUT \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  -d '{"url": "https://api.example.com/Scores/beforeDelete"}' \
+https://api.parse.com/1/hooks/triggers/Tournament/beforeDelete
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/triggers/Tournament/beforeDelete', json.dumps(
+      {"url": "https://api.example.com/Scores/beforeDelete"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{
+  "className": "Tournament",
+  "triggerName": "beforeDelete",
+  "url": "https://api.example.com/Tournament/beforeDelete",
+  "warning": "beforeDelete trigger already defined for class Tournament in cloud code"
+}
+```
+
+## Delete function webhook
+To delete a function webhook use the put method.
+
+```bash
+curl -X PUT \
+  -H "X-Parse-Application-Id: ${APPLICATION_ID}" \
+  -H "X-Parse-Master-Key: ${MASTER_KEY}" \
+  -H "Content-Type: application/json" \
+  -d '{"__op": "Delete"}' \
+https://api.parse.com/1/hooks/functions/foo
+```
+```python
+import json,httplib
+connection = httplib.HTTPSConnection('api.parse.com', 443)
+connection.connect()
+connection.request('PUT', '/1/hooks/functions/foo', json.dumps(
+      {"__op": "Delete"}
+      ), {
+       "X-Parse-Application-Id": "${APPLICATION_ID}",
+       "X-Parse-Master-Key": "${MASTER_KEY}",
+       "Content-Type": "application/json"
+     })
+result = json.loads(connection.getresponse().read())
+print result
+```
+
+The output may look like this:
+```json
+{}
+```
+
+If a cloud code trigger with the same name already exists then the it is returned as the result.
```

Comment:
s/trigger/function/
s/the//

Also add something like "Since the overriding webhook was just deleted, this cloud code function will be run from now on."